src/objective-c/GRPCClient/private/GRPCCompletionQueue.m

```diff
@@ -52,6 +60,7 @@ - (instancetype)init {
     // anymore (i.e. on self dealloc). So the block would never end if it
     // retained self.
     grpc_completion_queue *unmanagedQueue = _unmanagedQueue;
+    int64_t lTimeoutSecs = _timeoutSecs;
```

Comment:
We can just use `timeoutSecs` (the method argument) in the block, making the extra local variable unnecessary.